sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/BulkWriter.scala

```diff
@@ -555,9 +554,10 @@ private class BulkWriter(container: CosmosAsyncContainer,
     val bulkOperationResponseFlux: SFlux[CosmosBulkOperationResponse[Object]] =
       container
           .executeBulkOperations[Object](
-            bulkInputEmitter.asFlux().publishOn(bulkWriterBoundedElastic),
+            bulkInputEmitter.asFlux().publishOn(bulkWriterRequestsBoundedElastic),
```

Comment:
Shall we mention this change as well in the changelog?

Reply:
Done